src/mars_patcher/data/schema.json

```diff
@@ -467,6 +498,10 @@
             "minimum": 0,
             "maximum": 6
         },
+        "AreaIDKey": {
```

Comment:
Why not just use the existing AreaID def?

Reply:
Because this is used as a key. A key can only be of type string JSON.